karavan-web/karavan-app/src/main/webui/src/api/ProjectService.ts

```diff
@@ -258,7 +258,6 @@ export class ProjectService {
     public static copyProject(sourceProject: string, project: Project) {
         KaravanApi.copyProject(sourceProject, project, res => {
             if (res.status === 200 || res.status === 201) {
-                EventBus.sendAlert( 'Success', 'Project copied', 'success');
```

Comment:
@mgubaidullin Because with the current implementation it will get triggered even if the project doesn't actually get copied (because user sent an existing project-id) and this is a very confusing UX. That's why I mentioned that the best approach would be to have a UI validation, but that's a much bigger task and this was a quick fix to avoid issues.

Reply:
We can have UI validation because there might me project created by another user that is not yet sync with the current one, or not authorized to have access to. So, verification should be server-side. If copied than server returns 200 and we show notification, otherwise server return 409 and show error notofication

Reply:
I agree, but I again followed the implementation used in the create project action, which doesn't throw an exception if the existing project-id is sent, but just the sent project.
I can change this implementation and throw an exception when needed, but it won't follow the create project behaviour in that case.

Reply:
We could keep it as is and create an issues to reimplement for server validation both create and copy project

Reply:
@mgubaidullin My plan was to fix this quickly by just throwing the exception on back-end and handle it in the UI (as agreed in our previous comments) and after that implement the generic validation framework which I would apply to create/copy project actions initially, but could be used in the whole project.
I will have the PR for the quick fix at the beginning of the next week, since I'm busy this week.
Let me know if this is fine with you. If it is, please create a task for the generic validation mentioned above and assign it to me.